src/core/Alert/InlineAlert/InlineAlert.tsx

```diff
@@ -17,28 +17,26 @@ const inlineAlertClassNames = {
   smallScreen: `${baseClassName}--small-screen`,
 };
 
-export interface InlineAlertProps extends HtmlDivWithRefProps {
-  /** Style variant. Affects color and icon used.
+export interface InlineAlertProps extends HtmlDivProps {
+  /** Style variant. Affects color and icon.
    * @default 'neutral'
    */
   status?: 'neutral' | 'warning' | 'error';
   /** Main content of the alert */
   children?: ReactNode;
-  /** Use small screen styling */
+  /** Toggles small screen styling */
   smallScreen?: boolean;
   /** Label for the alert */
   labelText?: ReactNode;
-  /** Set aria-live mode for the alert text content and label.
+  /** Aria-live mode for the alert's text content and label. Not recommended to turn completely off
```

Comment:
This could have additional info underlining why it is not recommended to turn off

Reply:
This is now addressed in the latest commit